runtimes/rust/lbr-prelude-derive/build.nix

```diff
@@ -9,7 +9,7 @@
           extraSources = [
             {
               name = "lbr-prelude";
-              path = ../lbr-prelude;
+              path = config.packages.lbr-prelude-rust-src;
```

Comment:
nice! does rust-src contain all the 'extraSources` and `extraData`?

Reply:
We need to be careful, because at the moment two packages A and B use `extraSources = [C]`, that might be a problem?

Reply:
rust-src does not include other dependencies, they have to be included in the build.nix, and located in the .extras folder. This is to avoid problems with circular dependencies